NitroxServer/ConsoleCommands/DirectoryCommand.cs

```diff
@@ -8,24 +9,25 @@
 
 namespace NitroxServer.ConsoleCommands
 {
-    internal sealed class DirectoryCommand : Command
+    internal class DirectoryCommand : Command
     {
         public override IEnumerable<string> Aliases { get; } = new[] { "dir" };
 
-        public DirectoryCommand() : base("directory", Perms.CONSOLE, "Opens the directory of the main program")
+        public DirectoryCommand() : base("directory", Perms.CONSOLE, "Opens the current directory of the server")
         {
         }
 
         protected override void Execute(CallArgs args)
         {
             string dir = Path.GetDirectoryName(Assembly.GetEntryAssembly()?.Location);
+
             if (!Directory.Exists(dir))
             {
-                Log.Error($"Unable to open Nitrox directory '{dir}' because it does not exist.");
-                return;
+                throw new ArgumentException($"Unable to open Nitrox directory '{dir}' because it does not exist.");
             }
 
             Process.Start(dir);
+            Log.Info($"Opening directory '{dir}'");
```

Comment:
Maybe we should add a `dir` parameter to `Log.sensitiveLogParameters`. I thinking about some users who have their full name as their pc account and depending on the path it could be saved to the log file.

Reply:
Yes, would recommend rename to `path` and then check if the PC user name is in the path and redact it away.